easybuild/tools/repository/filerepo.py

```diff
@@ -109,7 +109,11 @@ def add_easyconfig(self, cfg, name, version, stats, previous):
         else:
             statstxt = statscomment + statsprefix + stats_to_str(stats, isyeb=yeb_format) + statssuffix
 
-        txt += statstxt
+        try:
+            txt += statstxt
+        except UnicodeDecodeError as e:
+            raise EasyBuildError("Got a unicode error here: type(txt) %s type(statstxt) %s", type(txt).__name__, type(statstxt).__name__)
```

Comment:
doesn't type return a Type object? I had some trouble with this when trying to debug, but it might have been something else.